Misc/python.man

```diff
@@ -164,10 +164,13 @@ for the named module and runs the corresponding
 file as a script.
 .TP
 .B \-O
-Turn on basic optimizations.  Given twice, causes docstrings to be discarded.
+Remove assert statements and any code conditional on the value of
+__debug__; augment the filename for compiled (bytecode) files by
+adding .opt-1 before the .pyc extension.
 .TP
 .B \-OO
-Discard docstrings in addition to the \fB-O\fP optimizations.
+Do \fB-O\fP and also discard docstrings; change the filename for
```

Comment:
Cheryl, can you explain the \fB, \fP markup and verify that you somehow tested that it does 'the right thing'?  I looked at both the web page and python -h output.

Reply:
When I changed the line, I kept that part from the original line and simply add the other text around it.  However, it appears the Github does attempt to show what will happen with it.  If you click on the 'View' button for this source, Github applies the markup in non-raw mode.  This shows up as bolded as does some of the text in the `-i` and `-I` option that has similar (though not exactly the same) markup.

Reply:
Thanks.  Enough to comfortably merge without personal test.